browser/extensions/api/moonpay_api.cc

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/extensions/api/moonpay_api.h"
+
+#include <string>
+#include <memory>
+#include <utility>
+
+#include "brave/browser/profiles/profile_util.h"
+#include "brave/components/moonpay/browser/regions.h"
+#include "brave/components/moonpay/common/pref_names.h"
+#include "brave/components/ntp_widget_utils/browser/ntp_widget_utils_region.h"
```

Comment:
this looks like it needs to be in a buildflag guard - https://github.com/brave/brave-core/pull/6681/files#diff-bbe23e77bde2e5441102b094d6214d82R115

Reply:
no longer relevant after removing ntp_widget buildflag